src/app/routes.ts

```diff
@@ -0,0 +1,13 @@
+//routes generator
+export function getSlidesRoutes(SlidesComponent, route?:string){
```

Comment:
call it rootRoute or something? 
Are you sure we need this at all? 
Doesn't it do the same thing as forChild?